include/oneapi/dpl/experimental/kt/esimd_radix_sort_onesweep.h

```diff
@@ -20,6 +20,79 @@
 
 namespace oneapi::dpl::experimental::esimd::impl
 {
+// float, 32bit
+template <typename T, bool __is_ascending, std::enable_if_t<std::is_same<T, float>::value && sizeof(T) == sizeof(::std::uint32_t), int> = 0>
+constexpr T
+__full_sort_identity()
```

Comment:
What do you think about moving this implementation into the utils header alongside with `__sort_identity`? 
I also suggest modifying the original `__sort_identity` so that it is a callable, and renaming `__full_sort_identity` to `__sort_identity` so it is just a specialization.